### PR TITLE
SMP-921: Integración del módulo de dispositivos en el plugin de Cordova

### PR DESCRIPTION
**Pull request by Jaquilar**
…lugin de Cordova

- Implemented the Devices module into the plugin

### CODE REVIEW

**Review by ArianaQ1 — APPROVED** [review]

File: src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.apache.cordova.CallbackContext
+import org.apache.cordova.plugin.sentinel.interfaces.ModuleDelegate
+
+class DevicesModule : ModuleDelegate {
```
Comment (brodriguezQXD) [inline src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt:14]: Missing `executeAction` ? Also all functions should be private

**Review by Smorales0492 — APPROVED** [review]

File: src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt (hunk 1 above)
Comment (Jaquilar) [reply src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt:14]: Done

File: src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    private val cloudProvider = SentinelProvider.cloudProvider
+    private val deviceServices = cloudProvider.devicesCloudServices
+
+    override fun executeAction(
+        action: String,
+        args: JSONArray,
+        callbackContext: CallbackContext,
+    ): Boolean {
+        val result =
+            when (action) {
```
Comment (brodriguezQXD) [inline src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt:25]: do `return when....`

**Review by brodriguezQXD — APPROVED** [review]

File: src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt (hunk 2 above)
Comment (Jaquilar) [reply src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt:25]: task/implement-device-manager-module branch got this change

Done

**Pull request merged**